indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -96,6 +96,16 @@
       list-style-type: none;
     }
   }
+
+  .checkbox-with-text {
+    input[type='checkbox'] {
+      margin-left: 1px;
+    }
+
+    div {
+      margin-left: 21px;
+    }
+  }
```

Comment:
I think the only thing we need is a small margin-right on the checkbox; the div isn't needed at all